toolkits/slack/arcade_slack/constants.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+class PositiveInt(int):
```

Comment:
constants.py might not be the best place for this. Maybe models.py or types.py ?

Reply:
yea, moved to models already